lib/commands/general.js

```diff
@@ -184,6 +184,12 @@ commands.mobileBackdoor = async function (opts = {}) {
   return await this.espresso.jwproxy.command(`/appium/execute_mobile/backdoor`, 'POST', {target, methods, elementId});
 };
 
+commands.uiautomator = async function (opts = {}) {
+  const {byMethod, value, action} = assertRequiredOptions(opts, ['byMethod', 'value', 'action']);
+  const {index} = opts;
+  return await this.espresso.jwproxy.command(`/appium/execute_mobile/uiautomator`, 'POST', {byMethod, value, index, action});
+};
```

Comment:
Would you have time to write a Javascript e2e test for this?

It wouldn't need to be anything major, just something that performs a very basic UiAutomator2 operation.

Let me know if you need help with that.

Reply:
@dpgraham I will be able to do that :)